src/routes/auth/invite-user.e2e.test.js

```diff
@@ -0,0 +1,99 @@
+const { faker } = require('@faker-js/faker')
+const { expect } = require('chai')
+const { fixtures } = require('../../../test/utils')
+
+const endpointRoute = '/api/v1/auth/invite-user'
+
+describe('POST /invite-user', () => {   
+	let nonAdminUser
+	let adminUser
+	let nonAdminAccessToken
+	let adminAccessToken
+	let invitedUserInfo
+
+	before(async () => {
+		nonAdminUser = await fixtures.insertUser({
+			role : 'user'
+		}) 
+		const body = { id: nonAdminUser.id, email: nonAdminUser.email }
+		nonAdminAccessToken = fixtures.generateAccessToken(
+			{user : body}
+		)
+		adminUser = await fixtures.insertUser({
+			role : 'admin'
+		}) 
+		const result = { id: adminUser.id, email: adminUser.email }
+		adminAccessToken = fixtures.generateAccessToken(
+			{user : result}
+		)
```

Comment:
cool, so we can simply pass on the data without  having a new variable. 

Additionally, I feel the name `result` doesn't give much of an explanation of the variable's purpose

```suggestion
		adminAccessToken = fixtures.generateAccessToken(
			{user :  { id: adminUser.id, email: adminUser.email }}
		)
```

Reply:
Thank you